### PR TITLE
DOC: Update references to the PDF specification

### PR DESCRIPTION
**Pull request by j-t-1**
Also change TABLE to table

### CODE REVIEW

File: docs/user/pdf-version-support.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ PDF comes in the following versions:
 * 2001: 1.4
 * 2003: 1.5
 * 2004: 1.6
-* 2006 - 2012: 1.7, ISO 32000-1:2008
```
Comment (stefan6419846) [inline docs/user/pdf-version-support.md:12]: Why dropping the ISO standard here?

File: docs/user/robustness.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Robustness and strict=False
 
-PDF is [specified in various versions](https://www.pdfa.org/resource/pdf-specification-index/).
-The specification of PDF 1.7 has 978 pages. This length makes it hard to get
+PDF is [specified in various versions](https://pdfa.org/resource/pdf-specification-archive/).
+The specification of PDF 2.0 has 1,003 pages. This length makes it hard to get
```
Comment (stefan6419846) [inline docs/user/robustness.md:4]: Do we really need the thousands divider here?

File: pypdf/constants.py [hunk 3]
```diff
@@ -212,11 +213,7 @@ class StreamAttributes:
 
 
 class FilterTypes:
-    """
-    Table 4.3 of the 1.4 Manual.
-
-    Page 354 of the 1.7 Manual
-    """
+    """§7.4  of the 1.7 and 2.0 references."""
```
Comment (stefan6419846) [inline pypdf/constants.py:216]:
```suggestion
    """§7.4 of the 1.7 and 2.0 references."""
```

File: pypdf/constants.py [hunk 4]
```diff
@@ -229,7 +226,7 @@ class FilterTypes:
 
 
 class FilterTypeAbbreviations:
-    """Table 4.44 of the 1.7 Manual (page 353ff)."""
+    """§8.9.7  of the 1.7 and 2.0 references."""
```
Comment (stefan6419846) [inline pypdf/constants.py:229]:
```suggestion
    """§8.9.7 of the 1.7 and 2.0 references."""
```

File: pypdf/constants.py [hunk 5]
```diff
@@ -264,7 +261,7 @@ class CcittFaxDecodeParameters:
 
 
 class ImageAttributes:
-    """Table 4.39 Pdf Reference 1.7 page 340+"""
+    """§11.6.5  of the 1.7 and 2.0 references."""
```
Comment (stefan6419846) [inline pypdf/constants.py:264]:
```suggestion
    """§11.6.5 of the 1.7 and 2.0 references."""
```

File: pypdf/constants.py [hunk 6]
```diff
@@ -544,7 +541,7 @@ class GraphicsStateParameters:
 
 
 class CatalogDictionary:
-    """Table 3.25 in the 1.7 reference."""
+    """§7.7.2  of the 1.7 and 2.0 references."""
```
Comment (stefan6419846) [inline pypdf/constants.py:544]:
```suggestion
    """§7.7.2 of the 1.7 and 2.0 references."""
```

**Comment by stefan6419846** [comment]
Could you please fix the merge conflicts?

File: docs/user/pdf-version-support.md (hunk 1 above)
Comment (j-t-1) [reply docs/user/pdf-version-support.md:12]: Not sure is needed, but fine to change to "2006: 1.7, ISO 32000-1:2008"

File: docs/user/robustness.md (hunk 2 above)
Comment (j-t-1) [reply docs/user/robustness.md:4]: No, fine to be 1003

**Comment by j-t-1** [comment]
Completely confused by this process. I must have done something wrong yesterday, as today's commit says 

> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

and has code outside of what I was changing.

**Comment by stefan6419846** [comment]
@j-t-1 Doesn't look https://github.com/py-pdf/pypdf/pull/2638/files correct? It's just the merge commit which shows the behavior you describe, but this should not really be an issue if the overall diff looks correctly as we are squashing your commits anyway.

**Comment by j-t-1** [comment]
> @j-t-1 Doesn't look https://github.com/py-pdf/pypdf/pull/2638/files correct? It's just the merge commit which shows the behavior you describe, but this should not really be an issue if the overall diff looks correctly as we are squashing your commits anyway.

I did a new one, please could you delete or close this one? I think this one was started in a detached state or something like that.

**Comment by stefan6419846** [comment]
Closing as duplicate of #2640.

**Pull request closed without merging**